RTN-004.tex

```diff
@@ -145,263 +177,257 @@ \subsubsection{Benefits}\label{sssec:intro_del_bene}
+\subsection{Selection Algorithm}\label{ssec:sel_alg}
 
+The groups described in Section \ref{ssec:sel_grps} reserve a portion of the total DP0 slots for applicants with different levels of expertise and experience with Rubin Observatory or similar projects (including novice-level), and for applicants from Chile. 
+A diverse population of delegates by career stage, science interest, and from groups that are underrepresented in astronomy on the basis of gender, race, disability, sexual orientation, etc. is also desired for DP0. 
 
-\subsection{Application Form Contents}\label{ssec:sel_form}
+To achieve this, a simple selection algorithm was developed in which every applicant is assigned a weight factor, based on their career stage, science interest, and underrepresented identity.
+The weight values are proportional to the fraction of applicants with those characteristics divided by a target fraction for that demographic. 
+Target fractions for: (1) career stage are based on promoting junior researchers; (2) science interest are approximately flat across each of the four LSST science pillars and interests outside the pillars; and (3) underrepresented identities are based on the US population.
 
-A diversity-based selection process requires that adequate appropriate information be collected from applicants.
-The information on the application form will also be used by the CET to help design the delegate resources described in Sections 4.2 and 4.3. 
+For each of the Groups A through G (in order; Section \ref{ssec:sel_grps}), a weighted random selection is made from the pool of delegates that indicated membership in the group.
+If an applicant was a member of multiple groups, after they are selected they are removed from the pool for the other groups (i.e., they cannot be selected more than once).
+This selection process is done 1000 times, with applicants scoring a point every time they're selected.
+A score-ranked list of applicants is then used to identify delegates and a waitlist.
+The Jupyter Notebook that ran this selection algorithm for DP0.1 is available at \url{ls.st/Document-37722}.
 
-The application form should be self-explanatory -- applicants should not \emph{have to} refer to this document (or any other) in order to complete the form -- but the form should also link to additional information about DP0.
-The application form should include a preamble with the deadline for consideration, the types of data included in DESC DC2, and information for a point-of-contact in case of questions.
+A very similar selection algorithm will be adopted for DP0.2, if the applications are oversubscribed.
+In the case of significant undersubscription, all applicants would be allocated an account.
+In the case of significant undersubscription {\it and} a lack of diversity in the applicant pool, a campaign of targeted advertising or participation invitations will be designed and deployed.
```

Comment:
In this case, would all applicants would be allocated an account , and then a campaign of targeted advertising be undertaken? Or would you run the targetted campaign first and see if you can get up to or above subscriotion limits and then run the selection algorithm?

Reply:
Clarified the plan:

A very similar selection algorithm will be adopted for DP0.2, if the applications are oversubscribed.
In the case of significant undersubscription from any targeted demographic (such as the delegate Groups, career stage, science interest, or underrepresented identity) or if there is a general lack of diversity in the applicant pool, some accounts will remain unassigned and a campaign of targeted advertising or participation invitations will be designed and deployed.
However, if the applications are undersubscribed but the pool is diverse, all applicants would be allocated an account.